pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: surf_lint_rules
-version: 1.5.0
+version: 2.0.0
 description: Lint rules for "Surf" company for Dart and Flutter projects.
 repository: "https://github.com/surfstudio/flutter-surf-lint-rules"
 issue_tracker: "https://github.com/surfstudio/flutter-surf-lint-rules/issues"
 
 dependencies:
-  dart_code_metrics: ^4.0.0-dev.2
-  flutter_lints: ^1.0.4
+  dart_code_metrics: ^5.5.1
```

Comment:
suggest removing the dependency
wht do u think?

Reply:
lets go merge this pr.

Reply:
I suggest make two independent versions of linter: pure Dart and with DCM

Reply:
@mpkander why?